bitnami/wordpress/Chart.yaml

```
@@ -31,4 +31,4 @@ name: wordpress
 sources:
   - https://github.com/bitnami/bitnami-docker-wordpress
   - http://www.wordpress.com/
-version: 10.3.0
+version: 10.3.1
```

```suggestion
version: 10.4.0
```

As this change implements a new functionality and not a fix to an existing one, this should be a minor instead of a patch version.